app/src/docs/v1.api-spec.yaml

```diff
@@ -497,28 +497,19 @@ components:
       type: object
       required:
         - name
-        - endpoint
-        - healthCheck
-        - authenticated
-        - authorized
+        - healthy
+        - info
       properties:
         name:
           type: string
-          description: The name of the endpoint
-          example: Example API
-        endpoint:
-          type: string
-          description: Base endpoint URL
-          example: 'https://example.com/v1/'
-        healthCheck:
-          type: boolean
-          description: Is the endpoint reachable
-        authenticated:
+          description: The name of the dependency
+          example: smtp
+        healthy:
           type: boolean
-          description: Are credentials valid to access endpoint
-        authorized:
-          type: boolean
-          description: Do the credentials have the right permissions
+          description: Is the dependency reachable/alive
+        info:
+          type: string
+          description: Any extra information, generally useful when dependency is unreachable
```

Comment:
healthy and info parameters should have example strings attached.

Reply:
healthy is a boolean, but adding example for info.